library/include/ck/library/host_tensor/host_utility.hpp

```diff
@@ -0,0 +1,17 @@
+#pragma once
+#include <vector>
+
+namespace ck {
```

Comment:
this function doesn't need to stay in ```ck``` namespce

Reply:
I will send another PR